certified-connectors/tegolySIGN/apiProperties.json

```diff
@@ -0,0 +1,52 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "a2b9e741-b3c9-49ab-b278-1f2c21eacf5c",
```

Comment:
@JanGosda please replace the actual data with dummy data "\<<Enter your client ID\>>"